src/main/user-impl/java/com/mysql/cj/jdbc/ha/plugins/failover/FailoverConnectionPlugin.java

```diff
@@ -1211,16 +1169,23 @@ private void processFailoverFailure(String message) throws SQLException {
   }
 
   private void setClusterId(String host, int port) {
+    final String clusterID;
     if (!StringUtils.isNullOrEmpty(this.clusterIdSetting)) {
-      this.topologyService.setClusterId(this.clusterIdSetting);
+      clusterID = this.clusterIdSetting;
+      metricsContainer.setClusterID(clusterID);
+      this.topologyService.setClusterId(clusterID);
     } else if (this.isRdsProxy) {
       // Each proxy is associated with a single cluster so it's safe to use RDS Proxy Url as cluster identification
-      this.topologyService.setClusterId(host + ":" + port);
+      clusterID = host + ":" + port;
+      metricsContainer.setClusterID(clusterID);
```

Comment:
Can we stay consistent with all the method names? We have a `setClusterID` and a `setClusterId`.
Can we use `setClusterId` for everything?